### DISCUSSION TITLE
企业微信 图文消息 怎么传参

### DISCUSSION THREAD

**Discussion by wy0115** [w7corp/easywechat#2099] [Q&A]
News（）   参数应该怎么传

**Comment by overtrue** [comment]
企业微信不适用

**Comment by lucky33015** [comment]
**传参格式 picurl参数可不传递,如果需要传递的话,去请求 [素材管理接口](https://work.weixin.qq.com/api/doc/90000/90135/90256)获得的url就是这里要传递的picurl**
![image](https://user-images.githubusercontent.com/25861419/132538075-8536265b-6e1c-4555-a63c-52dfcbddf45f.png)

**调用easywechat的NewItems来处理**
`new NewsItem($this->message)`

**发送成功效果**

![image](https://user-images.githubusercontent.com/25861419/132538498-afdc6276-054f-40f9-b6a9-c124c6009242.png)
